Sources/MapboxCoreNavigation/PredictiveCacheOptions.swift

```diff
@@ -0,0 +1,48 @@
+import MapboxNavigationNative
+
+/**
+ `PredictiveCacheOptions` controls various configurations for a `Predictive Caching` mechanic.
+ */
+public class PredictiveCacheOptions {
```

Comment:
Looks like we aren’t using reference semantics anywhere with this class, so we can make it a struct.